.gitlab-ci.yml

```diff
@@ -85,10 +85,12 @@ early_reindex:
   when: manual
   script:
     - make reindex
+  timeout: 1h 10m
```

Comment:
This might need an explanatory comment. This timeout is coupled with another timeout in another location. That's a brittle coupling which should be documented at least. In both places.